### PR TITLE
Add CORS policy to make the sample works with our front-end samples

### PR DESCRIPTION
**Pull request by laura-rodriguez**

### CODE REVIEW

File: resource-server/okta-aspnetcore-webapi-example/Controllers/MessagesController.cs [hunk 1]
```diff
@@ -2,32 +2,36 @@
...
-            return Json( new
+
+            return Json(new
             {
                 messages = new dynamic[]
                 {
                     new { Date = DateTime.Now, Text = "I am a Robot." },
                     new { Date = DateTime.Now, Text = "Hello, world!" },
-                }
+                },
```
Comment (OkGoDoIt) [inline resource-server/okta-aspnetcore-webapi-example/Controllers/MessagesController.cs]: Why is there a comma here?

File: resource-server/okta-aspnetcore-webapi-example/Controllers/MessagesController.cs [hunk 2]
```diff
@@ -2,32 +2,36 @@
...
 namespace okta_aspnetcore_webapi_example.Controllers
+#pragma warning restore SA1300 // Element should begin with upper-case letter
 {
     [Produces("application/json")]
     [Authorize]
     public class MessagesController : Controller
     {
         [HttpGet]
         [Route("~/api/messages")]
+        [EnableCors("AllowAll")]
```
Comment (OkGoDoIt) [inline resource-server/okta-aspnetcore-webapi-example/Controllers/MessagesController.cs]: Did we determine if we actually need both this and the `app.UseCors("AllowAll");` in startup.cs?

File: resource-server/okta-aspnetcore-webapi-example/Controllers/MessagesController.cs (hunk 1 above)
Comment (laura-rodriguez) [reply resource-server/okta-aspnetcore-webapi-example/Controllers/MessagesController.cs]: There's a Stylecop rule for this: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1413.md

File: resource-server/okta-aspnetcore-webapi-example/Controllers/MessagesController.cs (hunk 2 above)
Comment (laura-rodriguez) [reply resource-server/okta-aspnetcore-webapi-example/Controllers/MessagesController.cs]: Good call! ```app.UseCors("AllowAll")``` applies the CORS policy to every request. I'm gonna push an update for this.

**Comment by OkGoDoIt** [comment]
So I did some research and it turns out since C# 4.0, it's legal to include a trailing comma in collection initializers.  Weird, I never knew that.  Still feels like bad style to me, but if it's consistent with what we're doing elsewhere then I guess it's fine.

**Pull request merged**